generator/transit_generator.cpp

```diff
@@ -44,6 +48,22 @@ using namespace std;
 
 namespace
 {
+bool CompStopsById(Stop const & s1, Stop const & s2)
```

Comment:
LessById

Reply:
Renamed. Or... Do you propose to use my::LessBy instead? But in that case I'll have to copypast the the name of param m_id: `my::LessBy(&Stop::m_id)`.